### PR TITLE
PLF-8528:Truncate job title when popover limit reached

### PR DESCRIPTION
**Pull request by Jihed525**

### CODE REVIEW

File: webapp/resources/src/main/webapp/javascript/eXo/UIUserProfilePopup.js [hunk 1]
```diff
@@ -294,12 +294,17 @@
 
                  tdProfile.append(aProfile);
 
+                 if(json.position){
```
Comment (ahamdi) [inline webapp/resources/src/main/webapp/javascript/eXo/UIUserProfilePopup.js]: Does this fix the issue as proposed in https://jira.exoplatform.org/browse/PLF-8528?focusedCommentId=511347&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-511347

Comment (Jihed525) [reply webapp/resources/src/main/webapp/javascript/eXo/UIUserProfilePopup.js]: Sorry for this late answer,the proposed changes indeed fix this issue as [proposed](https://jira.exoplatform.org/browse/PLF-8528?focusedCommentId=511347&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-511347) as displayed in this [screenshot](https://jira.exoplatform.org/secure/attachment/122294/Screen%20Shot%202020-04-15%20at%209.36.24%20PM.png )

Comment (ahamdi) [reply webapp/resources/src/main/webapp/javascript/eXo/UIUserProfilePopup.js]: Ok could you include the content of the first 
```
 if(json.position)
```
inside the later one

Comment (Jihed525) [reply webapp/resources/src/main/webapp/javascript/eXo/UIUserProfilePopup.js]: > Ok could you include the content of the first inside the later one

Done

**Review by ahamdi — APPROVED** [review]
LGTM

**Pull request merged**